archie-utils/src/main/java/com/nedap/archie/xml/JAXBUtil.java

```diff
@@ -60,10 +66,32 @@ public static synchronized JAXBContext createAOMContext() {
     public static synchronized JAXBContext createRMContext() {
         try {
             List<Class<?>> classes = Lists.newArrayList(ArchieRMInfoLookup.getInstance().getRmTypeNameToClassMap().values());
+            removeClasses(classes);
```

Comment:
Shouldn't this call be in `createAOMContext` instead? Or both? And maybe `initArchieJaxbContext`?